binaryio/test/float.rkt

```diff
@@ -0,0 +1,41 @@
+#lang racket/base
+(require rackunit
+         binaryio/float
+         racket/math
+         math/flonum)
```

Comment:
This adds a dependence on package `math-lib`. Add `math-lib` to `deps` in `binaryio/info.rkt`. (Or avoid the dependence and define some/all of the constants locally. Either way is fine.)

Reply:
I added it to the `deps`. I took a quick look at the source for the constants it wouldn't be trivial to move them over independently as they're defined using some other internal functions of `math/flonum`.